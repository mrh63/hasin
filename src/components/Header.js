import React from 'react';
import TopHeader from './TopHeader';
import { Container, Navbar, Nav, NavItem, NavLink } from 'reactstrap';
import { NavLink as RRNavLink } from 'react-router-dom';

const Header = () => {

    return (
        <div className="header">
            <TopHeader />
            <div className="bottom-header">
                <Container className="d-flex justify-content-between">
                    <img src="/images/Logo-white.png" alt="logo" />
                    <Navbar expand="md">
                        <Nav className="mr-auto" navbar>
                            <NavItem>
                            <NavLink to="/"  tag={RRNavLink}>خانه</NavLink>
                            </NavItem>
                            <NavItem>
                                <NavLink to="/about"  tag={RRNavLink}>درباره ما</NavLink> 
                            </NavItem>
                        </Nav>
                    </Navbar>
                </Container>
            </div>
        </div>
    )
}

export default Header;