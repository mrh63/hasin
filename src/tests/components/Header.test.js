import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('should render Header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});

test('should Header have two link', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.find('NavLink').length).toBe(2);
});