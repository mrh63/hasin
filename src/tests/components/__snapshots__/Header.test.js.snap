// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="header"
>
  <TopHeader />
  <div
    className="bottom-header"
  >
    <Container
      className="d-flex justify-content-between"
      tag="div"
    >
      <img
        alt="logo"
        src="/images/Logo-white.png"
      />
      <Navbar
        expand="md"
        tag="nav"
      >
        <Nav
          className="mr-auto"
          navbar={true}
          tag="ul"
          vertical={false}
        >
          <NavItem
            tag="li"
          >
            <NavLink
              tag={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "displayName": "NavLink",
                  "propTypes": Object {
                    "activeClassName": [Function],
                    "activeStyle": [Function],
                    "aria-current": [Function],
                    "className": [Function],
                    "exact": [Function],
                    "innerRef": [Function],
                    "isActive": [Function],
                    "location": [Function],
                    "onClick": [Function],
                    "replace": [Function],
                    "strict": [Function],
                    "style": [Function],
                    "target": [Function],
                    "to": [Function],
                  },
                  "render": [Function],
                }
              }
              to="/"
            >
              خانه
            </NavLink>
          </NavItem>
          <NavItem
            tag="li"
          >
            <NavLink
              tag={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "displayName": "NavLink",
                  "propTypes": Object {
                    "activeClassName": [Function],
                    "activeStyle": [Function],
                    "aria-current": [Function],
                    "className": [Function],
                    "exact": [Function],
                    "innerRef": [Function],
                    "isActive": [Function],
                    "location": [Function],
                    "onClick": [Function],
                    "replace": [Function],
                    "strict": [Function],
                    "style": [Function],
                    "target": [Function],
                    "to": [Function],
                  },
                  "render": [Function],
                }
              }
              to="/about"
            >
              درباره ما
            </NavLink>
          </NavItem>
        </Nav>
      </Navbar>
    </Container>
  </div>
</div>
`;
